src/main/scala/org/broadinstitute/dsde/firecloud/service/TrialService.scala

```diff
@@ -429,36 +425,32 @@ final class TrialService
     }
   }
 
-  private def createBillingReport(requestContext: RequestContext, userInfo: UserInfo): Future[PerRequestMessage] = {
-    val properties: SpreadsheetProperties = makeSpreadsheetProperties("Trial Billing Project Report")
-    val sheet: JsObject = googleDAO.createSpreadsheet(requestContext, userInfo, properties)
-    Try(sheet.fields.getOrElse("spreadsheetId", JsString("")).asInstanceOf[JsString].value) match {
-      case Success(spreadsheetId) if spreadsheetId.nonEmpty =>
-        updateBillingReport(requestContext, userInfo, spreadsheetId)
-      case Failure(e) =>
-        logger.error(s"Unable to create new google spreadsheet for user context [${userInfo.userEmail}]: ${e.getMessage}")
-        throw new FireCloudExceptionWithErrorReport(ErrorReport(StatusCodes.InternalServerError, e.getMessage))
-    }
-  }
+  /** This method operates with service account credentials and can thus be invoked by anyone.
+    * As the developer, be careful about when you call this.
+    */
```

Comment:
I considered making an entirely separate Actor and moving these methods into the new actor, but it didn't seem worth it.